searx/engines/www1x.py

```diff
@@ -38,13 +38,14 @@ def request(query, params):
 def response(resp):
     results = []
     xmldom = etree.fromstring(resp.content)
-    xmlsearchresult = eval_xpath_getindex(xmldom, '//searchresult', 0)
+    xmlsearchresult = eval_xpath_getindex(xmldom, '//data', 0)
     dom = html.fragment_fromstring(xmlsearchresult.text, create_parent='div')
-    for link in eval_xpath_list(dom, '/div/table/tr/td/div[2]//a'):
+    for link in eval_xpath_list(dom, '//a'):
         url = urljoin(base_url, link.attrib.get('href'))
         title = extract_text(link)
-        thumbnail_src = urljoin(gallery_url, eval_xpath_getindex(link, './/img', 0).attrib['src'])
-
+        thumbnail_src = urljoin(
+            gallery_url, (eval_xpath_getindex(link, './/img', 0).attrib['src']).removeprefix(base_url)
```

Comment:
String method removeprefix is not available in python 3.8 / could you please modify it / thanks!

Reply:
Done. Can you check?